express-routes/server.ts

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res, next) => {
+  console.log('The date is', new Date(), req.method, req.path);
+  next();
+});
+
+app.get('/', (req, res, next) => {
+  res.send('Receiving Get response');
+  next();
```

Comment:
If you send a response, don't call `next()`.

The purpose of `next()` is to tell Express that you weren't able to complete the request so the next middleware in the list should give it a try. Since you did complete the request, you shouldn't call `next()`.